xbmc/video/VideoDatabase.cpp

```diff
@@ -4602,11 +4617,24 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE tvshow ADD userrating integer");
     m_pDS->exec("ALTER TABLE musicvideo ADD userrating integer");
   }
+
+  if (iVersion < 97)
+  {
+    m_pDS->exec("ALTER TABLE movie ADD votes integer");
+    m_pDS->exec("ALTER TABLE episode ADD votes integer");
+    m_pDS->exec("ALTER TABLE tvshow ADD votes integer");
+    m_pDS->exec("ALTER TABLE musicvideo ADD votes integer");
+    std::string sql = PrepareSQL("UPDATE movie SET votes = CAST(REPLACE(c%02d,',','') as INTEGER)", VIDEODB_ID_VOTES);
```

Comment:
I'm not sure if this is good enough. I have votes that look like `12,345` and others that look like `12'345` and I'm sure that with other languages/locales you'll also see other formats which are not covered by this.